libpod/image/pull.go

```diff
@@ -225,6 +225,9 @@ func (i *Image) pullImage(ctx context.Context, writer io.Writer, authfile, signa
 			images = append(images, imageInfo.image)
 		}
 	}
+	if len(images) == 0 {
+		return nil, errors.Errorf("unable to find image")
+	}
 	return images, errors.Wrapf(err, "error pulling image from")
```

Comment:
Not related to this PR, but I think this line should return a nil error. It doesn't make sense to return images and an error, where all the errors are already being caught in previous statements.

Reply:
We shouldn't be doing that check here. If we want to verify the registries.conf file exists, we should do it in NewRuntime as we first set up libpod.